modules/core/src/main/scala/org/tessellation/cli/method.scala

```diff
@@ -122,6 +124,7 @@ object method {
     environment: AppEnvironment,
     snapshotConfig: SnapshotConfig,
     seedlistPath: Option[Path],
+    stateChannelSeedlistPath: Path,
```

Comment:
Isn't it `seedListPath === stateChannelSeedlistPath` for Global L0?

Reply:
No, state channel seedlist path is a list of DAGAddresses.

Reply:
But isn't it still `seedList.map(_.toAddress) === stateChannelSeedlist` for Global L0?